recipe/meta.yaml

```diff
@@ -13,18 +13,21 @@ source:
 
 build:
   skip: true  # [win]
-  number: 1
+  number: 2
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
+    - {{ compiler('fortran') }}
```

Comment:
I'm not sure this is really needed :-/

Reply:
I'm afraid it does, is it a problem to keep this in?

Reply:
Not a problem. Just wanted to be sure a fortran compiler was needed. This is good to merge!